server/src/test/java/org/apache/iotdb/db/integration/IoTDBLoadExternalTsfileIT.java

```diff
@@ -202,8 +209,8 @@ public void loadSequenceTsfileTest() throws SQLException {
           StorageEngine.getInstance().getProcessor(new PartialPath("root.vehicle"))
               .getSequenceFileTreeSet());
       File tmpDir = new File(
-          resources.get(0).getTsFile().getParentFile().getParentFile().getParentFile(),
-          "tmp" + File.separator + new PartialPath("root.vehicle") + File.separator + "0");
+          resources.get(0).getTsFile().getParentFile().getParentFile().getParentFile().getParentFile(),
```

Comment:
Consider using `getSGByEngingFile` in `StorageEngine`.

Reply:
We can't use that because we have to find the parent dir of storage group rather than dir of storage group